test/Controller/AboutPeopleControllerTest.php

```diff
@@ -20,6 +20,199 @@ public function it_displays_the_people_page()
+                'http://api.elifesciences.org/subjects/subject',
+                ['Accept' => 'application/vnd.elife.subject+json; version=1']
+            ),
+            new Response(
+                200,
+                ['Content-Type' => 'application/vnd.elife.subject+json; version=1'],
+                json_encode([
+                    'id' => 'subject',
+                    'name' => 'Subject',
+                    'impactStatement' => 'Subject impact statement.',
+                    'aimsAndScope' => [
+                        [
+                            'type' => 'paragraph',
+                            'text' => 'Subject aims and scope.',
+                        ],
+                        // Only a single paragraph is supported on the people page.
+                        [
+                            'type' => 'paragraph',
+                            'text' => 'Subject aims and scope more.',
+                        ],
+                    ],
+                    'image' => [
+                        'banner' => [
+                            'uri' => 'https://www.example.com/iiif/image',
+                            'alt' => '',
+                            'source' => [
+                                'mediaType' => 'image/jpeg',
+                                'uri' => 'https://www.example.com/image.jpg',
+                                'filename' => 'image.jpg',
+                            ],
+                            'size' => [
+                                'width' => 800,
+                                'height' => 600,
+                            ],
+                        ],
+                        'thumbnail' => [
+                            'uri' => 'https://www.example.com/iiif/image',
+                            'alt' => '',
+                            'source' => [
+                                'mediaType' => 'image/jpeg',
+                                'uri' => 'https://www.example.com/image.jpg',
+                                'filename' => 'image.jpg',
+                            ],
+                            'size' => [
+                                'width' => 800,
+                                'height' => 600,
+                            ],
+                        ],
+                    ],
+                ])
+            )
+        );
+
+        $this->mockApiResponse(
+            new Request(
+                'GET',
+                'http://api.elifesciences.org/people?page=1&per-page=1&order=asc&subject[]=subject&type=senior-editor',
+                ['Accept' => 'application/vnd.elife.person-list+json; version=1']
+            ),
+            new Response(
+                200,
+                ['Content-Type' => 'application/vnd.elife.person-list+json; version=1'],
+                json_encode([
+                    'total' => 0,
+                    'items' => [],
+                ])
+            )
+        );
+
+        $this->mockApiResponse(
```

Comment:
The `/people` mocking is not relevent to this couple of tests, so it can be extracted into a private method that does what `getUrl()` does. Could call it `mockPeopleApi()` or similar.